ods-devenv/scripts/aqua/install-aqua.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
I'd recommend at least `set -eu` on any script so that it fails on error. Enabling pipefail could also be considered.

Reply:
This task is currently used manually if you need to install aqua in a box, if that is required I'll add that setting to the script.